src/frontend/src/common/components/library/data_subview/index.module.scss

```diff
@@ -6,13 +6,15 @@
 .samplesRoot {
   align-items: stretch;
   flex-direction: column;
+  padding: $space-xl;
 
   .searchBar {
     display: flex;
     flex-direction: row;
+    align-items: center;
     justify-content: space-between;
     width: 100%;
-    margin-bottom: $space-l;
+    margin-bottom: $space-xxl;
```

Comment:
so it's a little strange rn, but the plan with styling was eventually to migrate everything over to emotion/ styled. this is totally cool for now, just letting you know!

Reply:
no problem, I can work on converting them in a follow up PR. Didn't want to do too much in one set of changes.